tests/test_api_auth.py

```diff
@@ -189,10 +184,6 @@ def test_01_auth_with_split(self):
                 self.assertEqual('Authentication failure. Wrong credentials',
                                  result['error']['message'], result)
             # the realm will be split from the login name
-            if six.PY2:
-                expected = "The user User(login=u'selfservice', " \
-                           "realm=u'realm3', resolver='') exists in NO resolver."
-            else:
                 expected = "The user User(login='selfservice', " \
```

Comment:
here also

Reply:
fixed in a1e752d6c5abb37fab930e65b3c64b3680e3f3f1.